selenium-system-test/src/test/scala/hts/pages/BankDetailsPage.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hts.pages
+
+import hts.browser.Browser
+import hts.utils.Configuration
+import org.openqa.selenium.WebDriver
+
+object BankDetailsPage extends Page {
+
+  val expectedURL: String = s"${Configuration.host}/help-to-save/enter-uk-bank-details"
+
+  override val expectedPageHeader: Option[String] = Some("Which UK bank account do you want us to pay your bonuses and withdrawals into?")
+
+  override val expectedPageTitle: Option[String] = Some("Which UK bank account do you want us to pay your bonuses and withdrawals into?")
+
+  def enterValidDetails()(implicit driver: WebDriver): Unit = {
```

Comment:
it would be more general to have a `enterDetails` function instead with the details being passed in so that the function can be reused to pass in invalid details if we ever need to. The `vals` in the `ValidBankDetails` object would then move to the calling code